static/src/javascripts/projects/commercial/modules/messenger/passback.ts

```diff
@@ -25,6 +20,17 @@ const outstreamMobile: [number, number] = [
 	adSizes.outstreamMobile.height,
 ];
 
+const getValuesForKeys = (
+	keys: string[],
+	valueFn: (key: string) => string[],
+): Array<[string, string[]]> => keys.map((key) => [key, valueFn(key)]);
+
+const getPassbackValue = (source: string): string => {
+	const isMobile = (getBreakpoint() as string).startsWith('mobile');
```

Comment:
`getBreakpoint` from detect-viewport is typed!

Reply:
Thanks have updated :)